ci/build_wheel_cpp.sh

```diff
@@ -4,24 +4,21 @@
 set -euo pipefail
 
 package_dir="python/librmm"
+wheel_dir=${RAPIDS_WHEEL_DIR:-"dist"}
```

Comment:
Maybe `/tmp/wheelhouse` so we have a known name.

Reply:
We can define `RAPIDS_WHEEL_OUTPUT_DIR` in the CI images. Match the naming we use for conda.